tools/roscreate/src/roscreate/roscreatepkg.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
The combination of removing the executable flag but adding a shebang line doesn't seem to make sense?

Reply:
Totally agreed, updated.